apps/storybook/stories/Namekit/RegistrationPriceInfo.stories.tsx

```diff
@@ -1,33 +1,39 @@
 import {
   PrimaryRegistrationStatus,
   SecondaryRegistrationStatus,
+  buildTimestamp,
+  now,
 } from "@namehash/ens-utils";
 import type { Meta, StoryObj } from "@storybook/react";
 import {
   RegistrationPriceInfo,
   RegistrationPriceInfoDisplayingFormat,
 } from "@namehash/namekit-react";
 
+const ELEVEN_DAYS_FROM_NOW_TIMESTAMP = buildTimestamp(
+  BigInt(new Date().getTime()) / 1005n,
```

Comment:
I'm really confused by the logic here. Why 11 days? Eleven days from now would be 11 days in the future, not in the past. Why aren't we using `now()` here? Why are we dividing by a magic number here? Etc..